x-pack/plugin/redact/src/main/java/org/elasticsearch/xpack/redact/RedactProcessor.java

```diff
@@ -55,6 +55,12 @@ public class RedactProcessor extends AbstractProcessor {
     private static final String DEFAULT_REDACTED_START = "<";
     private static final String DEFAULT_REDACTED_END = ">";
 
+    protected static final String REDACT_KEY = "_redact";
+    protected static final String IS_REDACTED_KEY = "_is_redacted";
```

Comment:
As I'm [adding those fields to the Elasticsearch specification](https://github.com/elastic/elasticsearch-specification/pull/2965), I'm wondering, why do we need underscores here? Under `_ingest`, we already have `timestamp` and `pipeline` that don't use underscores.

Reply:
Hi @pquentin , to my understanding, underscore is convention for ES managed/populated attributes. For example, [Grok processor](https://www.elastic.co/guide/en/elasticsearch/reference/current/grok-processor.html) adds similar field `_ingest._grok_match_index` with underscore in ingest metadata.

Reply:
OK, thank you.